@xen-orchestra/lite/src/components/Button.tsx

```diff
@@ -16,8 +16,8 @@ interface Effects {}
 
 interface Computed {}
 
-const Button = withState<State, Props, Effects, Computed, ParentState, ParentEffects>({}, ({ label, onClick }) => (
-  <button onClick={onClick}>{label}</button>
+const Button = withState<State, Props, Effects, Computed, ParentState, ParentEffects>({}, ({ children, onClick }) => (
+  <button onClick={onClick}>{children}</button>
 ))
```

Comment:
Forward any other `HTML button` prop to the `button` element would be a good idea ?
  Like we do for [rangeInput](https://github.com/vatesfr/xen-orchestra/blob/f7df86e7a2f3e9e462d12cc13c9f401f2bfbbf0b/%40xen-orchestra/lite/src/components/RangeInput.tsx#L5).

Reply:
Yes but this is out of this PR's scope, which already adds to many changes IMO, so I'll do it in another PR.